code-samples/eventing/bookstore-sample-app/frontend/client/components/CommentDisplay.js

```diff
@@ -0,0 +1,22 @@
+const CommentDisplay = ({ comment }) => {
+	// Assume receiving a comment object
+	return (
+		<div className='flex my-4 p-4 justify-center'>
+			{/* Customized darkmode not working as expected yet */}
+			<div className='comment-display w-full w-7/12 flex flex-row rounded-lg p-4 bg-gray-800 text-white dark:bg-white'>
+				<div className='flex items-center justify-center md:justify-start md:w-1/12'>
+					<img
+						src={comment.avatar}
+						alt='Avatar'
+						className='rounded-full w-10 h-10'
+					/>
+				</div>
+				<div className='md:w-1/12 text-sm text-gray-200'> {comment.time} </div>
+				<div className='md:w-8/12'> {comment.text} </div>
+				<div className='md:w-2/12'> {comment.emotion} </div>
+			</div>
```

Comment:
The text is not center-aligned. 
![CleanShot 2024-04-01 at 23 46 00@2x](https://github.com/knative/docs/assets/36619969/a7a5956e-0374-495f-ac75-c19e075cca98)

According to the design graph, it supposed to look like this, all the components are center-aligned:
![CleanShot 2024-04-01 at 23 46 50@2x](https://github.com/knative/docs/assets/36619969/91c0f161-b4b7-48a0-b080-fb7477330064)


Also the size of the text and emoji could be adjusted to make them look the same as the design graph.